src/test-setup.js

```diff
@@ -79,6 +79,24 @@ global.OCP = {
 }
 global.IS_DESKTOP = false
 
+/**
+ * Polyfill for Blob.prototype.arrayBuffer
+ * Required as jsdom breaks Nodejs's native Blob
+ * @see: https://github.com/jsdom/jsdom/issues/2555
+ */
```

Comment:
Sorry, coding in comments :D

Fix eslint issues:

```suggestion
/**
 * Polyfill for Blob.prototype.arrayBuffer
 * Required as jsdom breaks Nodejs's native Blob
 *
 * @see https://github.com/jsdom/jsdom/issues/2555
 */
```